bp-esa-frontend/src/components/dashboard.js

```diff
@@ -0,0 +1,68 @@
+import React, { Component } from 'react';
+import Filters from './filters';
+import ReportsTables from './reportsTables';
+import { Row, Col} from 'reactstrap';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faCheck, faTimes } from '@fortawesome/free-solid-svg-icons'
+
+class Dashboard extends Component {
+    renderTableData = () => {
+        const tickIcon = <FontAwesomeIcon icon={faCheck} />;
+        const crossIcon = <FontAwesomeIcon icon={faTimes} />;
+        const tableData = {
+            tableHead: ['PARTNER NAME', 'SLACK', 'FRECKLE', 'EMAIL'],
+            //Test data to simulate pagination
+            tableBody: [
```

Comment:
Nitpick: Freckle is misspelt throughout this array. 

Also, sync with the backend team and get the exact structure of the object that will be used to determine whether a partner has completed the various application related processes so that we can have mock data that is as close to what we will be working with - as opposed to hard coding the icons.

Reply:
Okay. I am working on implementing the above feedback